crontab.yml

```diff
@@ -30,6 +30,13 @@
       minute: 8
       job: '/usr/local/bin/instance-tags "Env=prod" && /usr/local/bin/instance-tags "controller=True" && output-on-error ~/backup_db_to_s3.sh'
 
+  - name: "Schedule 'Rebuild Staging DB' job"
+    cron:
+      name: "Rebuild Staging DB"
+      hour: 4
+      minute: 45
+      job: '/usr/local/bin/instance-tags "Env=prod" && /usr/local/bin/instance-tags "controller=True" && output-on-error ~/rebuild_staging_db.sh'
```

Comment:
@GeoWill We use this same pattern elsewhere, but just to break down exactly what's going on in this line:

- `/usr/local/bin/instance-tags "Env=prod"` - Only run this job in the prod environment - not staging
- `/usr/local/bin/instance-tags "controller=True"` - Only run this job on the 'controller' instance. When we deploy, we randomly pick one instance and tag it as 'controller' (if we're only on one instance, this instance will be the controller). This instance runs all the scheduled jobs so if we're deployed on multiple instances, we don't run the same job in lots of places and flood slack, create race conditions, etc
- `output-on-error` - This script invokes another script and suppresses the console output if it exits with 0 but prints if it exits with >0 this means we get an email if it fails but not if it works.
- `~/rebuild_staging_db.sh` - Finally, actually run the script